src/main/resources/de/theit/jenkins/crowd/CrowdSecurityRealm/CacheConfiguration/config.jelly

```diff
@@ -31,4 +31,7 @@ THE SOFTWARE.
     <f:entry field="ttl" title="${%crowd.cache.TTL}">
         <f:select default="300"/>
     </f:entry>
+    <f:entry title="${%crowd.useTokenCache}" field="useTokenCache">
+        <f:checkbox />
+    </f:entry>      
```

Comment:
Please move this entry bellow `use cache form`
It's confusing to have those two check boxes so separated :D 

Furthermore, if this is an SSO only caching its worth mentioning what are other caches in helper msg